addons/stock/models/stock_picking.py

```diff
@@ -812,32 +812,26 @@ def _get_overprocessed_stock_moves(self):
         )
 
     @api.multi
-    def _create_backorder(self, backorder_moves=[]):
```

Comment:
I know it's stupid, but you should keep it in the method parameters, to avodi breaking overrides. However, I don't see the point of using it in the method itself. Probably to remove in master branch